### ISSUE TITLE
Open background group (or individual tab) in new window

### ISSUE THREAD

**Issue by rgavs** [Quicksaver/Tab-Groups#18]
One feature I think tab groups could definitely use is the ability to open a background group in a new window directly. 
-I've never coded around this area, but when I have time around the holidays I'll fork it and mess around with it myself a bit.

**Comment by nico202** [comment]
Yeah this is cool, i do it manually when switching to multiple monitors

**Comment by maverick74** [comment]
If one press Shift+Link it already opens the link in a new Window (whit a new Group). This, however, is not a Background Window or Group

**Comment by WolfganP** [comment]
+1. It may be also worthy to have a command to open each group in a different new window (ie News, Development, Social Networks, ...) and the whole set to be retained on Tab Groups on window close (and be updated in case a new tab is opened / closed / navigated to other URL)
